src/main/java/frc/robot/Robot.java

```diff
@@ -46,16 +43,20 @@ public void robotInit() {
       Logger.addDataReceiver(new WPILOGWriter()); // Log to a USB stick ("/U/logs")
       Logger.addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
       new PowerDistribution(1, ModuleType.kRev); // Enables power distribution logging
-    } else if (IS_REPLAY) {
-      setUseTiming(false); // Run as fast as possible
-
-      String logPath =
-          LogFileUtil
-              .findReplayLog(); // Pull the replay log from AdvantageScope (or prompt the user)
-      Logger.setReplaySource(new WPILOGReader(logPath)); // Read replay log
-      Logger.addDataReceiver(
-          new WPILOGWriter(
-              LogFileUtil.addPathSuffix(logPath, "_sim"))); // Save outputs to a new log
+    } else {
+      try {
+        String logPath = LogFileUtil.findReplayLog();
+        if (logPath != null) {
```

Comment:
Checking if logPath is a valid filepath here rather than just non-null would probably avoid needing the try-catch block
java.io.File class has an exists() method that should work.

Reply:
I had this thought too, but the file path may be different depending on what computer is being used, and as far as I can tell, there is no way to find the file path the code is searching for.

Reply:
I do think that way would be better if we can figure out the file path consistantly